src/PIL/features.py

```diff
@@ -180,7 +180,14 @@ def get_supported_features() -> list[str]:
     """
     :returns: A list of all supported features.
     """
-    return [f for f in features if check_feature(f)]
+    supported_features = []
+    for f, (_, flag, _) in features.items():
+        if flag is True:
+            if check_module("webp"):
```

Comment:
What is purpose of this change? check_feature Will also check module

Reply:
It prevents `get_supported_features()` from raising a deprecation warning.

Reply:
I don't like `check_module("webp")` hardcoding here. We could either use path from Feature to check the module, or still use `check_feature` function and suppress depiction warning

Reply:
Ok, I've pushed a dynamic version, see what you think.